infra/build/functions/update_build_status.py

```diff
@@ -0,0 +1,132 @@
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+"""Cloud function to request builds."""
+import os
+import logging
+
+import requests
+
+import google.auth
+from googleapiclient.discovery import build
+from google.cloud import ndb
+
+import build_and_run_coverage
+import build_project
+import builds_status
+from datastore_entities import BuildsHistory
+from datastore_entities import Project
+
+# pylint: disable=line-too-long
+BADGE_IMAGES_BASE_URL = "https://raw.githubusercontent.com/google/oss-fuzz/master/infra/gcb/badge_images/"
```

Comment:
single quotes. also, to avoid the line-too-long lint error, break up the string like e.g.

```python
... = (
   'abc'
   'def'
)
```